python_modules/dagster-graphql/dagster_graphql/schema/partition_sets.py

```diff
@@ -133,6 +133,14 @@ class Meta:
     numPartitionsFailed = graphene.NonNull(graphene.Int)
 
 
+class GraphenePartitionKeyRange(graphene.ObjectType):
+    class Meta:
+        name = "PartitionKeyRange"
+
+    start = graphene.NonNull(graphene.String)
```

Comment:
instead of a string can this just be the timestamp?

Reply:
fine if not

Reply:
I think i'd prefer to keep it as a string, so we can generalize the partition key range for other types of partitions defs too